www/generate_benchmark_list.js

```diff
@@ -20,9 +20,19 @@ function loadTestListFromJSON(callback) {
 
 function generateTestListHTML(directory, tests) {
     tests.sort();
+    if (directory == "sql") {
+	prioritizeImportantTests(tests)
+    }
     return _.values(tests).map((v) => `<a class="testName" href="/plot.html?directory=${directory}&test=${v}">${v}</a>`).join("<br/>");
 }
 
+function prioritizeImportantTests(testList) {
+    importantTests = ["BenchmarkInsert1000Multinode_Cockroach-16", "BenchmarkInsert1000_Cockroach-16", "BenchmarkSort100000Limit10Distinct_Cockroach-16", "BenchmarkSort100000Limit10_Cockroach-16"]
```

Comment:
As mentioned I just picked a few randomly to be replaced with tests we considered important. I think I'm just going to go with the tests from an email that Peter sent out a while ago comparing our performance to postgres.